scripts/import.ts

```diff
@@ -47,77 +47,256 @@ const rel = (absolutePath: string): string => {
   return path.relative(process.cwd(), absolutePath);
 };
 
+const addGenerationComment = (fileContents: string) => dedent`
```

Comment:
It's not due to the changes of this PR, but I noticed that there is a warning because `await` is not defined inside of `async` function in L482 and L484. 
<img width="714" alt="await" src="https://github.com/user-attachments/assets/db4863bc-f884-4e49-b9f6-c8a4bb28cb41" />

Seems like there are some ways to fix it: https://stackoverflow.com/questions/66486903/top-level-await-expressions-are-only-allowed-when-the-module-option-is-set-t. We will also fix it?

Reply:
This is top-level await, which is supported in Node for a while now. Not sure why TypeScript is complaining about it here, we already have `module` set to "ESNext":

```
// tsconfig.node.json
"module": "ESNext", // The type of file to emit (CommonJS/ESM/etc.)
```

My guess is that in your IDE it's not picking up this tsconfig for some reason.